codegen/test/chart-envvars/templates/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
 {{- $painterImage := $painter.image }}
       - name: painter
         image: {{ $painterImage.registry }}/{{ $painterImage.repository }}:{{ $painterImage.tag }}
-        imagePullPolicy: {{ $painterImage.pullPolicy }}
+        imagePullPolicy: {{ $painterImage.pullPolicy }}        
```

Comment:
where is this extra whitespace coming from?

Reply:
Fixed! 👍